caffeinated/payment_methods/Paypal_Pro/help_tabs/payment_methods_overview_paypalpro.help_tab.php

```diff
@@ -1,61 +1,61 @@
-<p><strong><?php _e('PayPal Pro', 'event_espresso'); ?></strong></p>
+<p><strong><?php esc_html_e('PayPal Pro', 'event_espresso'); ?></strong></p>
 <p>
-    <?php _e('Adjust the settings for the PayPal Pro payment gateway.', 'event_espresso'); ?>
+    <?php esc_html_e('Adjust the settings for the PayPal Pro payment gateway.', 'event_espresso'); ?>
 </p>
 <p>
     <?php printf(
-        __('See %1$shere%2$s for list of currencies supported by Paypal Pro.', 'event_espresso'),
+        esc_html__('See %1$shere%2$s for list of currencies supported by Paypal Pro.', 'event_espresso'),
         "<a href='https://www.paypal.com/multicurrency'>",
```

Comment:
Missing some things from here
```suggestion
        "<a href='https://www.paypal.com/multicurrency'  target='_blank' rel='noopener noreferrer'>",
```